### PR TITLE
Inside DevOps with Dan Horrocks-Burgess

### PR DESCRIPTION
**Pull request by steve-fenton-octopus**
# About

This is an Inside DevOps post to add to the series.

# Release

No specific release.

# Blog image idea

I've included a picture to be used for the image. We normally knock out the background and show it using one of our gradients and blobs :)

# Prerequisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [guide for blog content basics](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content) and [writing educational content](https://www.octopus.design/932c0f1a9/p/137b06-educational-content---how-we-write-about-what-we-do).
- [x] I've checked the content using Grammarly (Octopus has a corporate license) and the [Hemingway app](https://hemingwayapp.com/) to remove typos, grammatical errors, and passive voice. 
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content/t/817249) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://www.octopus.design/932c0f1a9/p/5061d7-working-with-images).
- [x] I've added alt text for all the images I've included (alt text describes an image to people unable to see it, 125 characters max)
- [x] I've drafted social media posts for Twitter and LinkedIn and will post them for review in #topic-social-media.
- [x] I've dropped a note in Slack in #topic-content-marketing to let Tegan Ali know my post is ready for review. (No need to fill out a content brief.)

## How to review this PR

All the good stuff :)

## Publication date

Any time is fine.

### CODE REVIEW

File: blog/2023-11/inside-devops-dan-horrocks-burgess/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+**Can you recommend a good DevOps book?**
+
+*Dan:* Kanban From The Inside by Mike Burrows gives practical insights into the application of Kanban and the reasons behind the practices. It's a useful read for DevOps people as it covers things that happen prior to that all-important commit. It also has useful information for your continuous improvement process.
+
+**What's one thing about you that might surprise us?**
+
+*Dan:* I'm a huge Disney fan and I'm visiting 4 of the 6 global resorts this year: Paris, California, Florida, and Tokyo. I'm hoping to add Shanghai and Hong Kong later on.
+
+:::hint
+If you’d like to be featured in our series, [Inside DevOps](https://octopus.com/blog/tag/Inside%20DevOps), please reach out to [Joanna on LinkedIn](https://www.linkedin.com/in/joannawyganowska/) to set up a time for a quick chat.
```
Comment (TeganAli) [inline blog/2023-11/inside-devops-dan-horrocks-burgess/index.md:89]: @steve-fenton-octopus would you like me to add your name and LinkedIn profile here in addition to Joanna's?

Comment (steve-fenton-octopus) [reply blog/2023-11/inside-devops-dan-horrocks-burgess/index.md:89]: I think we'll leave Joanna's as Inside DevOps is her idea and I don't want to steal the credit. If we change the process we'll have to update all the Inside DevOps footers.

File: blog/2023-11/inside-devops-dan-horrocks-burgess/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+*Dan:* Initially, the focus of the DevOps movement was on fostering closer collaboration between development and operations teams. However, we soon recognized the importance of integrating other silos from business and security in the same way.
+
+Over the past 5 years, the trajectory of DevOps has shifted, emphasizing culture, team dynamics, and interactions rather than being solely tool-centric. The human aspect of DevOps is crucial as these factors allow technical practices to impact business outcomes.
+
+Companies who ask for help getting started with DevOps often want to treat it as they do when they engage an external DBA or security testing organization. They want someone to _build DevOps_ for them rather than make changes to their existing structures.
+
+You can't shut the door on a DevOps office, and hope they'll solve this problem. DevOps requires broad changes to the process, technology, team and interactions, and culture. Everyone has a part to play in that.
+
+**What are some DevOps best practices your organization has implemented?**
```
Comment (TeganAli) [inline blog/2023-11/inside-devops-dan-horrocks-burgess/index.md:42]: @steve-fenton-octopus is this the right question? Feels like Dan's answering a different question and talking generally about organizations that adopt DevOps. Unless he's saying that his organization has fallen short? 
I could be wrong, just want to check. Thanks :)

Comment (steve-fenton-octopus) [reply blog/2023-11/inside-devops-dan-horrocks-burgess/index.md:42]: I think the idea here is...

What are some best practices...

1. Lots of teams have done the basic best practices - source control and automated builds
2. What they aren't doing as much as test and deployment automation (so they should add them)
3. They should also join deployments to the original work items to give the business traceability

So, he's saying source control, build automation (most folks have done this) plus automated tests and deployments (fewer folks have done this) plus infrastructure automation (not implemented to the right level).

There may be a better way to say this.

Comment (TeganAli) [reply blog/2023-11/inside-devops-dan-horrocks-burgess/index.md:42]: @steve-fenton-octopus, perhaps the questions should be: What are some DevOps best practices you see people implementing?

Comment (steve-fenton-octopus) [reply blog/2023-11/inside-devops-dan-horrocks-burgess/index.md:42]: Yes! That makes sense as they consult with organizations.

File: blog/2023-11/inside-devops-dan-horrocks-burgess/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
```
Comment (TeganAli) [inline blog/2023-11/inside-devops-dan-horrocks-burgess/index.md:42]:
```suggestion
**What are some DevOps best practices you see organizations implementing?**
```

**Review by TeganAli — APPROVED** [review]

**Pull request merged**